### DISCUSSION TITLE
Playlist

### DISCUSSION THREAD

**Discussion by andercard0** [TeamNewPipe/NewPipe#11135] [Questions]
Hi.
Could you guys please consider to make Bookmarked playlists grid a bit bigger than it is atm? 

As a example:

Small and a bit tough to organize;

![20240604_192551.jpg](https://github.com/TeamNewPipe/NewPipe/assets/43047877/563991ed-b244-4f8a-a9ad-d27395436319)

As example as far as i remember the thumbnails are big and easy to seee.

![20240604_192657.jpg](https://github.com/TeamNewPipe/NewPipe/assets/43047877/5ae04c9d-b457-4413-9e24-3c873bac6c17)

**Comment by KitschCatch** [comment]
Also is there any chance we might get a shuffle feature?
